lib/agenda/index.ts

```diff
@@ -101,7 +101,7 @@ class Agenda extends EventEmitter {
     super();
 
     this._name = config.name;
-    this._processEvery = (humanInterval(config.processEvery) ?? humanInterval('5 seconds')) as number;
+    this._processEvery = (humanInterval(config.processEvery) ?? humanInterval('5 seconds')) as number; // eslint-disable-line @typescript-eslint/non-nullable-type-assertion-style
```

Comment:
This must be a new lint-rule that came with the latest dep-update.

Unfortunately I don't understand the rule. I also don't think it applies to this line.

Reply:
I had a quick look and didn't understand it either.

Added lint-ignore here too https://github.com/agenda/agenda/pull/1182/files#r549456554